capa/ghidra/capa_explorer.py

```diff
@@ -359,9 +363,26 @@ def main():
         popup("capa explorer found no matches.")  # type: ignore [name-defined] # noqa: F821
         return capa.main.E_EMPTY_REPORT
 
-    for item in parse_json(capa_data):
-        item.bookmark_functions()
-        item.label_matches()
+    user_choice = askChoice(  # type: ignore [name-defined] # noqa: F821
```

Comment:
Let's use the Ghidra FlatProgramAPI method [askChoices](https://ghidra.re/ghidra_docs/api/ghidra/app/script/GhidraScript.html#askChoices(java.lang.String,java.lang.String,java.util.List)) due to the combo of items we enable users to select. See my comments below for more on what I mean here.